index.js

```diff
@@ -43,7 +43,13 @@ SuperCluster.prototype = {
         this.points = points;
 
         // generate a cluster object for each point and index input points into a KD-tree
-        var clusters = points.map(createPointCluster);
+        var clusters = [];
+        points.forEach(function (point, i) {
```

Comment:
Avoid using `forEach` in performance-critical code.` for (var i = 0; i < points.length; i++)` is much faster.